src/web/aws-cloudwatch/KinesisSetup.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Button, Col } from 'react-bootstrap';
+import { Input } from 'components/bootstrap';
+
+export default class KinesisSetup extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    onChange: PropTypes.func.isRequired,
+    getValue: PropTypes.func.isRequired,
+  }
+
+  render() {
+    const { getValue, onChange, onSubmit } = this.props;
+
+    return (
+      <form onSubmit={onSubmit}>
+        <Col md={9} mdOffset={6}>
```

Comment:
The `Col` sizes are not adapted to the latest changes in Wizard, so the page is not centered.